src/Microsoft.XmlSerializer.Generator/tests/Microsoft.XmlSerializer.Generator.Tests.csproj

```diff
@@ -35,7 +35,7 @@
     </PropertyGroup>
     <Copy SourceFiles="$(ToolsDir)\csc.runtimeconfig.json" DestinationFiles="$(OutputPath)dotnet-Microsoft.XmlSerializer.Generator.runtimeconfig.json" />
     <Message Text="Running Serialization Tool" Importance="normal" />
-    <Exec ContinueOnError="true" Command="$(ToolsDir)\dotnetcli\dotnet $(OutputPath)dotnet-Microsoft.XmlSerializer.Generator.dll $(OutputPath)Microsoft.XmlSerializer.Generator.Tests.dll /force /casesensitive"  />
+    <Exec ContinueOnError="true" Command="$(RuntimePath)\dotnet $(RuntimePath)dotnet-Microsoft.XmlSerializer.Generator.dll $(OutputPath)Microsoft.XmlSerializer.Generator.Tests.dll /force /casesensitive"  />
```

Comment:
I don't know if that's the right dotnet to use.  You may want to just call $(TestHostExecutablePath) which should point at the same dotnet used by the tests.  @weshaggard what do you think?

Reply:
@ericstj Actually RuntimePath doesn't work. It gave the error that the required library hostfxr.dll could not be found at that location. But $(TestHostExecutablePath)  also not work, it gave the error " '\dotnet.exe' is not recognized as an internal or external command"

Reply:
For now you can just change the path to dotnet-Microsoft.XmlSerializer.Generator.dll.  That should get you unblocked.  You can look into running on the test's CLI instead of tool CLI as a seperate issue.  I'm not even 100% certain that's the right thing for you to do since I don't understand what type of app this is.